x-pack/platform/plugins/private/reindex_service/server/src/constants.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+//
+export const API_BASE_PATH_UPRGRADE_ASSISTANT = '/api/upgrade_assistant';
```

Comment:
am I understanding correctly that we are mounting the reindexing_service routes under `/api/upgrade_assistant`? I remember we briefly spoke about it, but cant remember what we ended up deciding

Reply:
Correct, to be addressed in a future PR.